ci-operator/config/openshift/release/openshift-release-master__ocp-4.7.yaml

```diff
@@ -190,6 +190,12 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: ipi-azure-arcconformance
+- artifact_dir: /tmp/artifacts
+  as: e2e-ovirt-upgrade
+  cron: 0 3,16,22 * * *
```

Comment:
Why the weird numbers?

Reply:
helps us sync the upgrade jobs,
They are the only jobs that doesn't use tmpfs so if they start all at once then they all use the same RHV storage domain which can result in etcd problems and failures on install

Reply:
What's wrong with standard 4hrs?

Reply:
If I'll specify each hours for example I'll get the jobs started at:4,8,12... and all the upgrade jobs will start at the same time which is very bad for us...

Reply:
You should tweak leases to prevent that

Reply:
Using leases will cause bigger problems :/
If they will all start at the same time and there will be 1 lease for a job then I'll have 3 upgrade jobs waiting for a lease, since an upgrade job takes around 2-2.5h that will mean that 2 jobs will timeout due to lease pending problems